MovingBlocks/Terasology
# Work on render nodes: exposure and bloom

### Contains
- Continues refactoring work of the renderer, to take advantage of the new node-based architecture.
- See commit descriptions below for an overview of the changes

### How to test
- Run Terasology
- Create a new game or load an existing one
- Verify nothing has changed in the rendering, especially when activating/deactivating Eye Adaptation and Bloom.

### Notes:
- While the bloom effect is in now taking advantage of the new architecture... in practice it doesn't seem to work. This seems to be a pre-existing issue however and I won't worry about fixing it now.
- In general a number of problems have crept into the renderer while the overhaul is taking place no matter our best intention to keep everything working. 
- These problems will be solved during planned additional refactoring passes through all the nodes in the renderer. These refactoring passes are anyway needed as work on the later nodes has shown ways to improve the code in earlier nodes.
- Furthermore, an overhaul of the visual debug system (showing the content of a number of FBOs) is also planned and will help pinpoint problems that have visual consequences.
